pkg/k8s/run.go

```diff
@@ -18,6 +18,17 @@ import (
 
 // Run runs scan on kubernetes cluster
 func Run(cliCtx *cli.Context) error {
+	// Full-cluster scanning with '--format table' without explicit '--report all' is not allowed so that it won't mess up user's terminal.
+	if cliCtx.String("report") == "all" &&
+		!cliCtx.IsSet("report") &&
```

Comment:
isn't the conditions `cliCtx.String("report") == "all" &&  !cliCtx.IsSet("report")` contrudict?

Reply:
We want `all` to be passed explicitly when someone is scanning a full cluster or namespace, even though it is the default.

Reply:
@josedonizetti I think we should add more explanation so that other developers won't be confused.

Reply:
👍